.github/workflows/migrations-check.yml

```diff
@@ -30,41 +30,23 @@ jobs:
           git merge --no-ff --no-log origin/master
       - name: Create dirs and files
         run: |
-          mkdir -p /home/runner/work/albs-web-server/alts
-          mkdir -p /home/runner/work/albs-web-server/albs-frontend
-          mkdir -p /home/runner/work/albs-web-server/albs-node
-          mkdir -p /home/runner/work/albs-web-server/albs-sign-node
-          mkdir -p /home/runner/work/albs-web-server/alma-tests-cacher
-          touch /home/runner/work/albs-web-server/alma-tests-cacher/vars.env
-          mkdir -p /home/runner/work/albs-web-server/albs-sign-file
-          touch /home/runner/work/albs-web-server/albs-sign-file/.env
-      - name: Add env file
-        run: |
```

Comment:
These vars are moved to the `tests/test-vars.env` file. It's symlinked on line 35.